lib/matplotlib/_mathtext.py

```diff
@@ -570,7 +570,7 @@ def __init__(self, default_font_prop: FontProperties, load_glyph_flags: LoadFlag
 
         super().__init__(default_font_prop, load_glyph_flags)
         for texfont in "cal rm tt it bf sf bfit".split():
-            prop = mpl.rcParams['mathtext.' + texfont]
+            prop = mpl.rcParams['mathtext.' + texfont]  # type: ignore[index]
```

Comment:
There is no getter for a group so I think ignoring the error is the only solution.

Reply:
I assume mypy is complaining here. Would it help to make the list explicit?

```
for texfont ["cal", "rm", "tt", "it", "bf", "sf", "bfit"]:
```

Reply:
Ok in that case, ignoring is likely the simplest solution.